docs/artifact_repositories/npm.md

```diff
@@ -30,7 +30,7 @@ Custom NPM registries can be configured with `npm`:
 
 ```sh
 npm config set replace-registry-host never
-npm config set registry https://<PHYLUM_GROUP>:<PHYLUM_API_KEY>@npm.phylum.io/
+npm config set registry https://<PHYLUM_ORG>%2F<PHYLUM_GROUP>:<PHYLUM_API_KEY>@npm.phylum.io/
```

Comment:
I don't see why it would need to be explained. The same argument could be made about the `:`. If you don't know what something is for, all you need to do is google it to figure it out. I don't see why anyone would just omit things they don't understand.